plugins/Makefile

```diff
@@ -996,8 +996,24 @@ PLUGIN_FILES += $(filter-out Sapphire/src/plugin.cpp,$(wildcard Sapphire/src/*.c
 PLUGIN_FILES += $(filter-out sonusmodular/src/sonusmodular.cpp,$(wildcard sonusmodular/src/*.cpp))
 
 # --------------------------------------------------------------
-# stocaudio
+# Starling Via
+
+PLUGIN_FILES += $(wildcard StarlingVia/src/*.cpp)
+PLUGIN_FILES += $(wildcard StarlingVia/Via/io/src/*.cpp)
+PLUGIN_FILES += $(wildcard StarlingVia/Via/ui/src/*.cpp)
+PLUGIN_FILES += $(wildcard StarlingVia/Via/modules/*/*.cpp)
+PLUGIN_BINARIES += StarlingVia/res/original.gateseq
```

Comment:
are these binaries MIT licensed too? if not it needs special mentioning on the LICENSES.md file

Reply:
These binaries are generated by this tool, which doesn't have an explicit license yet: https://github.com/starlingcode/viatools
This project isn't officially completed, as mentioned here: https://github.com/starlingcode/Via-for-Rack/issues/31#issuecomment-1777974374

It would be good to have something like a CC0 license for those binaries to make sure they can be distributed.
MIT wouldn't make much sense since there is no "source" in this case. These are not executable binaries as discussed previously, but of a custom filetype used just for the Via project.

Reply:
I am okay with this, those files are alike music resources (for example wavetable or midi) which are binary in nature but doesn't mean they are executable code.